pkg/cmd/pulumi/new_test.go

```diff
@@ -1187,3 +1179,11 @@ func TestPulumiPromptRuntimeOptions(t *testing.T) {
 	require.Equal(t, 1, len(proj.Runtime.Options()))
 	require.Equal(t, "someValue", proj.Runtime.Options()["someOption"])
 }
+
```

Comment:
Can you revert this back to where it was? Seems unneeded to move it, just introduces blame noise

Reply:
Ach yep this was a mistake. Will do.